src/App.test.tsx

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import axios from 'axios';
+import App,{Breed, getBreeds, GetBreedsResponse} from './App';
 
-test('renders learn react link', () => {
+describe("app",()=>{
+
+it('should renders hello world', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/hello world/i);
   expect(linkElement).toBeInTheDocument();
 });
+it('should renders a list of cat breeds', () => {
+  render(<App />);
+  const listElement = screen.getAllByRole('listitem');
+  expect(listElement).toHaveLength(3);
+});
+
+});
+
+describe("api",()=>{
+jest.mock("axios");
+it('Should return a list with 3 cat breeds when API/breeds call is successful', async ()=> {
+const mockBreeds: GetBreedsResponse = {
+  data: [
+    {
+      name: 'ragdol'
+    },
+    {
+      name: 'persa'
+    },
+    {
+      name: 'srd'
+    }] 
+  };
+  
+  (axios.get as jest.Mock).mockResolvedValueOnce(mockBreeds);
```

Comment:
Talvez vc ainda tenha um problema de typeScript, mas nesse caso vai ser alguma proproedade que vai estar faltando no seu dado mockado. AI é só ir completando na linha 24.

```suggestion
  mockedAxios.get.mockResolvedValue(mockBreeds);
```

Reply:
deu certo desse jeito :)